.abf.yml

```diff
@@ -1,2 +1,2 @@
 sources:
-  cdrtools-3.01.tar.bz2: 62766aa9bfeec91b3c78da7c59b5c96bfbec84d9
+  cdrtools-3.02a01.tar.bz2: a7fccb3b16c7a4fc49d1926d72f20ebba290029a
```

Comment:
Indeed, I changed this manually. I though the 2 files (cdrtools.spec and .abf.yml) had to be changed with a single commit. And I also did not know about the filestore and "abf put". I haven't read the docs yet and cannot promisse I'll have time to do so in the near future. So please feel free to do whatever you wish with this PR. Thanks in advance if you can do something with my diff. Cheers